kong/workspace_service_test.go

```diff
@@ -32,6 +32,10 @@ func TestWorkspaceService(T *testing.T) {
 	assert.Nil(err)
 	assert.NotNil(workspace)
 
+	exists, err = client.Workspaces.Exists(defaultCtx, createdWorkspace.ID)
+	assert.Nil(err)
+	assert.True(exists)
```

Comment:
Jumped the gun on the initial review; realized that the Enterprise test failures were actually legit:
```
kong/workspace_service_test.go:35:2: undefined: exists
kong/workspace_service_test.go:37:14: undefined: exists
```
```suggestion
	var exists bool
	exists, err = client.Workspaces.Exists(defaultCtx, createdWorkspace.ID)
	assert.Nil(err)
	assert.True(exists)
```